### ISSUE TITLE
Error using --match-edid

### ISSUE THREAD

**Issue by alkim0** [phillipberndt/autorandr#273]
This is the traceback:
```
home: renaming display DisplayPort-5 to DisplayPort-4
Unhandled exception (dictionary keys changed during iteration). Please report this as a bug at https://github.com/phillipberndt/autorandr/issues.
Traceback (most recent call last):
  File "/usr/bin/autorandr", line 1562, in <module>
    exception_handled_main()
  File "/usr/bin/autorandr", line 1546, in exception_handled_main
    main(sys.argv)
  File "/usr/bin/autorandr", line 1347, in main
    update_profiles_edid(profiles, config)
  File "/usr/bin/autorandr", line 652, in update_profiles_edid
    for c in profile_config.keys():
RuntimeError: dictionary keys changed during iteration
```
Changing line 652 from `for c in profile_config.keys():` to `for c in list(profile_config.keys()):` works.

**Comment by phillipberndt** [comment]
Thanks!

**Comment by phillipberndt** [comment]
Fix released as 1.12.1

**Comment by mattalexx** [comment]
Thank you thank you thank you! My output names are changing every reboot, and this means I can still use autorandr.
